String index out of bounds for variant GRCh37 `chr11:67258349AGCTGGACCCAGCC>GT`

Hi guys,
I am using Jannovar v0.32 to perform variant annotation and I am getting an error when I try to annotate variant `chr11:67258349AGCTGGACCCAGCC>GT`.

When I run
```bash
java -jar jannovar-cli-0.32.jar annotate-pos -d jannovar/v0.32/hg19_ensembl.ser -c 'chr11:67258349AGCTGGACCCAGCC>GT'
```

I get the following output:
```
Options
JannovarAnnotatePosOptions [genomicChanges=[chr11:67258349AGCTGGACCCAGCC>GT], toString()=JannovarAnnotationOptions [useThreeLetterAminoAcidCode=false, nt3PrimeShifting=false, showAll=false, databaseFilePath=jannovar/v0.32/hg19_ensembl.ser, toString()=JannovarBaseOptions [reportProgress=true, httpProxy=null, httpsProxy=null, ftpProxy=null, verbosity=1]]]
Deserializing transcripts...
INFO Deserializing JannovarData from jannovar/v0.32/hg19_ensembl.ser
INFO Deserialization took 8.65 sec.
#change	effect	hgvs_annotation	messages
[ERROR] Could not annotate variant chr11:67258349AGCTGGACCCAGCC>GT!
java.lang.StringIndexOutOfBoundsException: begin 229, end 233, length 232
	at java.base/java.lang.String.checkBoundsBeginEnd(String.java:3319)
	at java.base/java.lang.String.substring(String.java:1874)
	at de.charite.compbio.jannovar.annotation.builders.BlockSubstitutionAnnotationBuilder$CDSExonicAnnotationBuilder.<init>(BlockSubstitutionAnnotationBuilder.java:153)
	at de.charite.compbio.jannovar.annotation.builders.BlockSubstitutionAnnotationBuilder.build(BlockSubstitutionAnnotationBuilder.java:61)
	at de.charite.compbio.jannovar.annotation.builders.AnnotationBuilderDispatcher.build(AnnotationBuilderDispatcher.java:66)
	at de.charite.compbio.jannovar.annotation.VariantAnnotator.buildNonSVAnnotation(VariantAnnotator.java:149)
	at de.charite.compbio.jannovar.annotation.VariantAnnotator.buildAnnotations(VariantAnnotator.java:130)
	at de.charite.compbio.jannovar.cmd.annotate_pos.AnnotatePositionCommand.run(AnnotatePositionCommand.java:63)
	at de.charite.compbio.jannovar.Jannovar.main(Jannovar.java:67)
```

Any idea what's going on?

Thank you,
Daniel

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -14,6 +14,7 @@
 * Remove hard-coded chrMT renaming. Filenames for download that have a ? in their URL will be splitted and only the first part before the ? is used a file name in the download path.
 * Making faMT annotation for refSeq optional
 * Update RefSeq parser that does not run into null-pointer exceptions on mouse and rat genome (e.g. when no exon defines the gene name)
+* Fixing issue with block substitutions (#475).
 
 ### jannovar-htsjdk
 

--- a/jannovar-core/src/main/java/de/charite/compbio/jannovar/annotation/builders/BlockSubstitutionAnnotationBuilder.java
+++ b/jannovar-core/src/main/java/de/charite/compbio/jannovar/annotation/builders/BlockSubstitutionAnnotationBuilder.java
@@ -150,8 +150,8 @@ public final class BlockSubstitutionAnnotationBuilder extends AnnotationBuilder 
 			this.varChangeLastPos = varChangeLastPos;
 			// "(...+2)/3" => round up integer division result
 			this.aaChange = new AminoAcidChange(refChangeBeginPos.getPos() / 3,
-				wtAASeq.substring(refChangeBeginPos.getPos() / 3, (refChangeLastPos.getPos() + 1 + 2) / 3),
-				varAASeq.substring(varChangeBeginPos.getPos() / 3, (varChangeLastPos.getPos() + 1 + 2) / 3));
+				wtAASeq.substring(refChangeBeginPos.getPos() / 3, Math.min((refChangeLastPos.getPos() + 1 + 2) / 3, wtAASeq.length())),
+				varAASeq.substring(varChangeBeginPos.getPos() / 3, Math.min((varChangeLastPos.getPos() + 1 + 2) / 3, varAASeq.length())));
 
 			// Look for stop codon, starting at change position.
 			this.varAAStopPos = varAASeq.indexOf('*', refChangeBeginPos.getPos() / 3);

